src/app/admin/pages/reviewers/reviewers.component.scss

```diff
@@ -0,0 +1,187 @@
+@import '~_vars.scss';
+$light-blue-gradient: linear-gradient(45deg, saturate(darken($bright-blue, 10), 10), saturate(lighten($bright-blue, 10), 10));
+
+.cname {
+  color: $bright-blue;
+}
+
+.users {
+    max-width: 100%;
+    position: relative;
+    margin: auto;
+    box-sizing: border-box;
+    min-height: 500px;
+    margin-top: 20px;
+    margin-bottom: 10px;
+  }
+  
+  .user-cards {
+    margin-top: 30px;
+    display: grid;
+    grid-template-columns: repeat(6, minmax(10%, 300px)); 
+  }
+  
+  .user-cards > *:not(:last-child) {
+    margin-right: 30px;
+    margin-bottom: 30px;
+  }
+
+button {
+  font-size: 14px;
+  border: 0;
+  padding: 10px 20px 10px 25px;
+
+  &.good {
+    color: white;
+    background: $bright-blue;
+    border-color: $bright-blue;
+  }
+}
+
+.reviewer-search {
+  width: 550px;
+  margin-top: 5px;
+  z-index: 1;
+  position: absolute;
+  top: 70px;
+  right: 30px;
+}
+
+.reviewer-search__container {
+  padding: 10px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0 2px 6px 2px rgba(0, 0, 0, 0.2);
+}
+
+.container__user-dropdown {
+  position: relative;
+}
+
+.container__prompt {
+  text-align: center;
+  margin: 10px;
+}
+
+.container__prompt__loading {
+  margin-top: 10px;
+  text-align: center;
+  font-size: 14px;
+}
+
+.prompt__icon {
+  color: $bright-blue;
+  font-size: 40px;
+}
+
+.prompt__text {
+  color: $placeholder;
+  font-size: 20px;
+  margin-top: 5px;
+}
+
+.user-dropdown__search {
+  border: 1px solid $placeholder;
+  border-radius: 5px;
+  padding: 5px;
+  font-size: 17px;
+}
+
+.user-dropdown__search:focus-within {
+  border-color: $bright-blue;
+}
+
+.search__input {
+  width: 90%;
+  border: none;
+  outline: none;
+}
+
+.search__input::placeholder {
+  color: $placeholder;
```

Comment:
If I recall correctly, the `$placeholder` variable is sampled directly from the Chrome placeholder color value. Why is this here?

Reply:
@nrwinner maybe we should add a comment explaining that. I had no idea that was the case. Also, does that mean other browsers have a different color? Do we want that to fluctuate, or be consistent?

Reply:
What do you recommend I replace the use of `$placeholder` with?

Reply:
@DonSeannelly @nvisal1 Browsers do have slightly different placeholder values (they're all light gray though). That variable was intended for color-matching icons to input placeholders as closely as possible. That rule can be removed since we're taking the input for an element that's already styled by the browser and manually restyling it to essentially the same thing.

Reply:
@nrwinner we should definitely add a comment to the stylesheet on that. I had no idea that was the purpose, and I've definitely used it for things other than icons.

Reply:
Removing this rule makes the placeholder text much darker
<img width="665" alt="Screen Shot 2019-03-25 at 7 16 15 AM" src="https://user-images.githubusercontent.com/32280549/54915594-e6442a80-4ecd-11e9-9183-894ec2b673cd.png">
Thoughts?